src/encdatavault_fmt_plug.c

```diff
@@ -345,59 +389,129 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 		unsigned char result[ENC_KEY_SIZE * ENC_MAX_KEY_NUM] = { 0 };
 
 		// Key derivation based on MD5
-		if ((cur_salt->version >> 4) == 0) {
-			for (i = 0; i < MIN_KEYS_PER_CRYPT; ++i) {
-				tmp.u64[0] = 0;
-				tmp.u64[1] = 0;
+		for (i = 0; i < MIN_KEYS_PER_CRYPT; ++i) {
```

Comment:
Can simplify to assume `MIN_KEYS_PER_CRYPT` is 1 for MD5 for now (once you split the MIN/MAX between formats), or just revert to using old code (prior to the introduction of PBKDF2), although arguably written this way the code is slightly closer to what it'd be for SIMD support.

Reply:
If you don't mind I would like to keep it that way. I may try to support SIMD later.

Reply:
OK, keep it.